ext/pdo_oci/config.m4

```diff
@@ -177,31 +177,17 @@ if test "$PHP_PDO_OCI" != "no"; then
   dnl
   dnl Check if we need to add -locijdbc8
   dnl
-  PHP_CHECK_LIBRARY(clntsh, OCILobIsTemporary,
-  [
-    AC_DEFINE(HAVE_OCILOBISTEMPORARY,1,[ ])
-  ], [
+  PHP_CHECK_LIBRARY(clntsh, OCILobIsTemporary, [], [
     PHP_CHECK_LIBRARY(ocijdbc8, OCILobIsTemporary,
     [
       PHP_ADD_LIBRARY(ocijdbc8, 1, PDO_OCI_SHARED_LIBADD)
```

Comment:
I wonder whether this part is still necessary or not...

Reply:
Removed also, yes. This was also cleaned a bit more via b8e1d4d4e5af74a9d5a95d03adbe2b93e58301b7 for ext/oci8 extension but not for the pdo_oci.

Reply:
Fun side fact ext/oci8 should be then named ext/oci :D